### PR TITLE
updates the copyright cleanup script, runs it on all source

### PR DESCRIPTION
**Pull request by ecoon**
@lipnikov  This is run with the updated script.  I'm fairly sure this keeps all comments in blocks with COPYRIGHT or Author lines, while also removing all comment blocks that were empty once the copyright and author lines were removed.  This was a bit tricky with all combinations of "decorations" like "------" or "******" etc, but some spot checks suggest it worked.

### CODE REVIEW

File: config/SuperBuild/templates/unittest-TestTestMacros.cpp [hunk 1]
```diff
@@ -1,243 +1,252 @@
-#include "../UnitTest++.h"
```
Comment (lipnikov) [inline config/SuperBuild/templates/unittest-TestTestMacros.cpp:1]: All files in templates/ should be skipped.

**Comment by lipnikov** [comment]
@jd-moulton Copyright is slightly different. Do you approve it?

File: config/SuperBuild/templates/unittest-TestTestMacros.cpp (hunk 1 above)
Comment (ecoon) [reply config/SuperBuild/templates/unittest-TestTestMacros.cpp:1]: Ugh, I thought I only ran this in $AMANZI_SRC_DIR/src.  Will revert.

**Comment by ecoon** [comment]
> @jd-moulton Copyright is slightly different. Do you approve it?

It looks like the COPYRIGHT files should be updated in both Amanzi and ATS.  The Amanzi one does not include ORNL, the ATS one is only LANL.  

The statement I made was to make this generic and scalable to whichever institutions choose to participate in the future.  Obviously this can be changed too.

File: src/data_structures/CompositeVector.hh [hunk 2]
```diff
@@ -1,9 +1,15 @@
...
   Amanzi is released under the three-clause BSD License.
   The terms of use and "as is" disclaimer for this license are
   provided in the top-level COPYRIGHT file.
 
-  Author: Ethan Coon (ecoon@ornl.gov)
+  Authors: Ethan Coon (ecoon@ornl.gov)
+*/
+
+/*
+  Amanzi is released under the three-clause BSD License.
```
Comment (lipnikov) [inline src/data_structures/CompositeVector.hh:11]: Second copyright message

Comment (ecoon) [reply src/data_structures/CompositeVector.hh:11]: Yep, looks like some files had a blank line maybe?  Fixing this now and will rerun.

File: src/mesh/mesh_logical/MeshLogical.hh [hunk 3]
```diff
@@ -1,10 +1,15 @@
...
   Amanzi is released under the three-clause BSD License.
   The terms of use and "as is" disclaimer for this license are
   provided in the top-level COPYRIGHT file.
 
-  Author: Ethan Coon (ecoon@lanl.gov)
+  Authors: Ethan Coon (ecoon@lanl.gov)
+*/
+
+/*
+  Amanzi is released under the three-clause BSD License.
```
Comment (lipnikov) [inline src/mesh/mesh_logical/MeshLogical.hh:11]: Second copyright statement

**Review by lipnikov — APPROVED** [review]

**Comment by lipnikov** [comment]
David runs paperwork, so I leave Copyright statement to him. I skimmed randomly through 25% of files. Looks good to me.

File: src/data_structures/CompositeVector.hh (hunk 2 above)
Comment (ecoon) [reply src/data_structures/CompositeVector.hh]: new more aggressive approach gets this

**Comment by ecoon** [comment]
@jd-moulton  are you ok with this generic language in the files?  No changes have been made to the COPYRIGHT file, just to the header.

```  
 Copyright 2010-202x held jointly by participating institutions.
 Amanzi is released under the three-clause BSD License.
 The terms of use and "as is" disclaimer for this license are
 provided in the top-level COPYRIGHT file.
```

**Comment by jd-moulton** [comment]
@ecoon Yes, this generic language for the source code is good.   Any nuances in collaborating institutions can be covered in the actual COPYRIGHT file.

**Pull request merged**